Misc/NEWS.d/next/Core and Builtins/2021-09-09-15-05-17.bpo-45155.JRw9TG.rst

```diff
@@ -0,0 +1,2 @@
+int.to_bytes() now takes default arguments for ``length`` and ``byteorder``,
+defaulting to ``1`` and ``sys.byteorder`` respectively.
```

Comment:
This should mention both `` :meth:`int.to_bytes` `` and `` :meth:`int.from_bytes` ``